Python/Product/Analysis/Parsing/ParserOptions.cs

```diff
@@ -31,7 +31,7 @@ public sealed class ParserOptions {
 
         public Severity IndentationInconsistencySeverity { set; get; }
 
-        public bool Verbatim { get; set; } = true;
+        public bool Verbatim { get; set; } = false;
```

Comment:
What is affected by this change?

Reply:
EVERYTHING! This is what we suggested in response to your performance analysis finding the huge amount of time spent in GetNodeAttributes, as this should skip most of those.

The main thing it affects is anywhere we try to convert an AST branch back into code. Before, we had all the whitespace, etc. required to do it perfectly, but now we have to rely on formatting options (and we now have `CodeFormattingOptions.Traditional`, which is basically the normal style for Python code).

It was relevant here because we convert AST nodes into text for hover that we don't necessarily do elsewhere - default values for function parameters is the main one, IIRC. If we've parsed verbatim, then comments before or within a node will appear in the representation, which is not what we want. We only really want to do verbatim parsing explicitly when we know we are going to need to round-trip (i.e. formatting, extract method).